### PR TITLE
Suppresses scrolling of body while modal is displayed

### PR DESCRIPTION
**Pull request by nabeliwo**
The scroll is suppressed by giving a style to the body when active props of Box component becomes true.
Changing the style of the body may have an unexpected effect.

I want you to check carefully 🙏

### CODE REVIEW

File: src/components/Modal/Box.tsx [hunk 1]
```diff
@@ -20,21 +20,45 @@ interface MergedStyledProps extends InjectedProps {
...
-    {active ? (
-      <React.Fragment>
-        <Background {...props} onClick={hideModal} />
-        <Inner {...props}>{children}</Inner>
-      </React.Fragment>
-    ) : null}
-  </Wrapper>
-)
+interface State {
+  scrollTop: number
```
Comment (koba04) [inline src/components/Modal/Box.tsx]: I'm not sure why the `scrollTop` is required as a state.
Do you have any problems if you only apply `overflow: hidden` to the body without managing `scrollTop`?

Comment (nabeliwo) [reply src/components/Modal/Box.tsx]: If implemented with `overflow: hidden` only, it worked fine.
It seems that I was thinking too hard... 😭

**Comment by nabeliwo** [comment]
@koba04 @im36-123 Made the implementation simple.
Please review again 🙏

File: src/components/Modal/Box.tsx [hunk 2]
```diff
@@ -20,17 +20,15 @@ interface MergedStyledProps extends InjectedProps {
...
 
-const BoxComponent: React.FC<Props & InjectedProps> = ({
-  active,
-  children,
-  hideModal,
-  ...props
-}) => (
+const BoxComponent: React.FC<Props> = ({ active, children, hideModal, ...props }) => (
   <Wrapper className={active ? 'active' : ''} {...props}>
+    {console.log(active)}
```
Comment (koba04) [inline src/components/Modal/Box.tsx]: Lets' remove `console.log` 😄

Comment (nabeliwo) [reply src/components/Modal/Box.tsx]: oh...
Thank you!

**Pull request merged**